.github/workflows/build-package.yml

```diff
@@ -0,0 +1,19 @@
+name: Debian Package Build
+on:
+  pull_request:
+    branches:
+      - current
```

Comment:
The job should be triggered only on “equuleus”

Reply:
done